CHANGELOG.md

```diff
@@ -59,6 +59,7 @@
   * Allow uppercase characters in default return url scheme
 * ThreeDSecure
   * Add `setRequestedExemptionType` to `ThreeDSecureRequest`
+* *Please note:* This version is dependent on the Java 8 programming language. Please read [Use Java 8 language features](https://developer.android.com/studio/write/java8-support) in the Android developer guide to learn how to use it in your project. If this causes an issue with your integration, please contact our [support](https://developer.paypal.com/braintree/help) team for further assistance.
```

Comment:
Question - Are there some stats we can point to - such as EOL for Java 7, something like [this](https://www.oracle.com/java/technologies/java-se-support-roadmap.html) which shows extended support dropping in July 2022?

Basically - is there any Java timelines or Android documentation we can point to that reveals the age of Java 7, making it more easily justifiable for us dropping support in a minor version.

Reply:
Yeah the best thing I cold find was this [Android Gradle Plugin (AGP) Release Note from March 2021](https://developer.android.com/studio/past-releases/past-agp-releases/agp-4-2-0-release-notes#java-8-default). They made Java 8 the default. After this point we had to explicitly declare Java 7 support, but i'm sure that accelerated Java 8 adoption overall.